diesel_tests/tests/schema.rs

```diff
@@ -241,7 +237,7 @@ pub fn drop_table_cascade(connection: &TestConnection, table: &str) {
         .unwrap();
 }
 
-#[cfg(not(feature = "sqlite"))]
+#[cfg(feature = "postgres")]
```

Comment:
Is it right that this test isn't run for mysql any more?

Reply:
This isn't a test. It's a regular function that MySQL never calls

Reply:
This isn't a test. It's a regular function that MySQL never calls